pulp_container/app/redirects.py

```diff
@@ -132,3 +132,21 @@ def redirect_to_object_storage(self, artifact, return_media_type):
         }
         content_url = artifact.file.storage.url(artifact.file.name, parameters=parameters)
         return redirect(content_url)
+
+
+class GCloudStorageRedirects(S3StorageRedirects):
+    """
+    A class that implements methods for the direct retrieval of manifest objects.
+    """
+
+    def redirect_to_object_storage(self, artifact, return_media_type):
+        """
+        Redirect to the passed artifact's file stored in the Azure storage.
```

Comment:
s/Azure/GCP

Reply:
I think we have a pulpcore function for this now.